data/en/mbs_0117.json

```diff
@@ -0,0 +1,407 @@
+{
+    "title": "Monthly Business Survey",
+    "survey_id": "009",
+    "mime_type": "application/json/ons/eq",
+    "theme": "default",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
```

Comment:
eq_id and form_types need to be added

Reply:
Don't worry about this - only relevant for using quick-launch. Most surveys don't have these values and SDX are nearly ready for us to merge all of these to pre-prod so we can use the launcher form.